src/fsharp/tast.fs

```diff
@@ -3100,6 +3100,18 @@ and
     ///   - If this is an implementation of an abstract slot then this is the name of the property implemented by the abstract slot
     member x.PropertyName               = x.Deref.PropertyName
 
+    /// Indicates whether this value represents a property getter.
+    member x.IsPropertyGetter = 
```

Comment:
Please call this ``IsPropertyGetterMethod`` and ``IsPropertySetterMethod`` and adjust the identical code in ``Symbols.fs`` to use these.